test/models/frontend/ShortServiceRefundsAnswersSpec.scala

```diff
@@ -22,20 +22,20 @@ import testdata.shortServiceRefunds._
 
 class ShortServiceRefundsAnswersSpec extends AnyWordSpecLike {
 
-  "toOverseasPensions" should {
-    "convert ShortServiceRefundsAnswers to a valid OverseasPensionContributions" in {
-      val result = shortServiceRefundsAnswers.toOverseasPensions
-
-      assert(result == overseasPensionContributions)
-      assert(result.shortServiceRefund == BigDecimal(1.0))
-      assert(result.shortServiceRefundTaxPaid == BigDecimal(2.0))
-      assert(result.overseasSchemeProvider == Seq(ukOverseasSchemeProvider))
-    }
-  }
-
-  "convert an Empty ShortServiceRefundsAnswers to a valid Empty OverseasPensionContributions" in {
-    val result = ShortServiceRefundsAnswers().toOverseasPensions
-
-    assert(result == OverseasPensionContributions(Seq[OverseasSchemeProvider](), BigDecimal(0), BigDecimal(0)))
-  }
+//  "toOverseasPensions" should {
```

Comment:
let's add TODO if we plan to fix it in some point. In future we'll clean up all TODOs in the project.